common/src/main/java/com/codeforcommunity/enums/Country.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.enums;
+
+public enum Country {
+  COUNTRY_ONE("one"),
+  COUNTRY_TWO("two"),
+  COUNTRY_THREE("three"),
+  COUNTRY_FOUR("four"),
+  COUNTRY_FIVE("five"),
+  COUNTRY_SIX("six");
```

Comment:
We know what countries these will be 🤔

Reply:
What countries are they?

Reply:
Antigua and Barbuda, Dominica, Grenada, St. Kitts & Nevis, St. Lucia, St. Vincent and the Grenadines

Reply:
Thank you @jacktonina